### PR TITLE
ncurses: Fix for GCC 5.1

### PR DESCRIPTION
**Pull request by sjackman**
Fix the build for GCC 5.1
`error: expected ')' before 'int' in definition of macro 'mouse_trafo'`
See http://lists.gnu.org/archive/html/bug-ncurses/2014-07/msg00022.html
and http://trac.sagemath.org/ticket/18301
Disable linemarker output of cpp

### CODE REVIEW

**Comment by sjackman** [comment]
See https://gist.github.com/anonymous/3745eb9d09c23a85d41e#file-02-make-L56-L63
```
gcc-5 -DHAVE_CONFIG_H -I../ncurses -I.   -D_DARWIN_C_SOURCE -D_XOPEN_SOURCE_EXTENDED -DNDEBUG -I. -I../include -I/usr/local/Cellar/ncurses/5.9/include/ncursesw -O2 --param max-inline-insns-single=1200 -dynamic -c ../ncurses/./base/lib_hline.c -o ../obj_s/lib_hline.o
In file included from ../ncurses/curses.priv.h:283:0,
                 from ../ncurses/lib_gen.c:19:
_18682.c:1364:15: error: expected ')' before 'int'
../include/curses.h:1906:56: note: in definition of macro 'mouse_trafo'
 #define mouse_trafo(y,x,to_screen) wmouse_trafo(stdscr,y,x,to_screen)
                                                        ^
make[1]: *** [../obj_s/lib_gen.o] Error 1
```

File: ncurses.rb [hunk 1]
```diff
@@ -28,6 +28,13 @@ class Ncurses < Formula
   def install
     ENV.universal_binary if build.universal?
 
+    # Fix the build for GCC 5.1
+    # error: expected ')' before 'int' in definition of macro 'mouse_trafo'
+    # See http://lists.gnu.org/archive/html/bug-ncurses/2014-07/msg00022.html
+    # and http://trac.sagemath.org/ticket/18301
+    # Disable linemarker output of cpp
+    ENV.append("CPPFLAGS", "-P")
```
Comment (mistydemeo) [inline ncurses.rb]: Nitpick, but no parens here.

Comment (sjackman) [inline ncurses.rb]: No worries. I'm a style nut too.

**Comment by mistydemeo** [comment]
Thanks!

**Comment by sjackman** [comment]
Thanks for the quick merge, Misty.

**Pull request closed without merging**